fuel-forge/build.gradle.kts

```diff
@@ -1,7 +1,24 @@
-plugins { java }
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins { java
+    kotlin("jvm") version "1.3.0-rc-116"
```

Comment:
you don't need version one here because you already have gradle-plugin on the classpath.